taskcluster/kinds/inference/kind.yml

```diff
@@ -53,12 +53,12 @@ tasks:
         - bash
         - -c
         - >-
-            task inference-test
+            task inference-test-local
 
   build-wasm:
     description: "Build the wasm bindings for the inference engine"
     dependencies:
-      build-local: inference-build-local
+      build-local: inference-test-local
```

Comment:
What is this change for?

Reply:
I'm preemptively renaming inference-test to inference-test-local to make way for inference-test-wasm in a subsequent PR.

Reply:
Right, but here you're changing the `build-wasm` task's dependency away from `inference-build-local`, `inference-test` is not involved?

Reply:
Oh sorry, I actually misread which line you were asking about (still having my morning coffee). 

To me it made sense to not even build the WASM unless the C++ _tests_ are passing, rather than just the C++ build. 

Somewhat of a drive-by change, but I feel that it should have always been this from the start.

Reply:
I'm just going to move this change entirely to the other PR. Yesterday, I was thinking "I'll just make all the config changes in one patch." But this change is more related to the new tests than it is to the Dockerfiles. 

I understand your confusion on why it's here.